src/features/player/movement.js

```diff
@@ -206,8 +194,10 @@ export default function handleMovement(player) {
         return attemptMove("EAST");
       case "j":
         return attemptMove("SOUTH");
-      case "Escape":
-        return VimMoves();
+      case "Backspace":
+        if (store.getState().vimCommand.length === 1) {
+          return VimMoves();
+        } else { return }
```

Comment:
the length check === 1 means that if there was a single character in the box, and backspace is pressed, VimMoves is run. (so if it contained 'k' before it would still work)